README.md

```diff
@@ -240,43 +123,24 @@ Example:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/JOdiemo/emunerable/issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
The links to the `issues` and `license` pages are broken. Kindly make sure that you do not have any broken links in README to make it more professional.